components/BWSHead/index.js

```diff
@@ -24,33 +24,43 @@ class BWSHead extends React.Component {
           <title>{pageTitle}</title>
 
           <meta charSet="utf-8" />
+          <meta name='application-name' content='BWS Collection' />
+          <meta name="referrer" content="origin-when-cross-origin" />
 
-          <meta
-            name="viewport"
-            content="width=device-width, initial-scale=1, shrink-to-fit=no"
-          />
+          <meta name='apple-mobile-web-app-capable' content='yes' />
+          <meta name='apple-mobile-web-app-status-bar-style' content='default' />
+          <meta name='apple-mobile-web-app-title' content='BWS Collection' />
```

Comment:
Should we spell out `Black Women's Suffrage`? I think we should if there's room.